pkg/render/monitor.go

```diff
@@ -162,6 +166,36 @@ func (mc *monitorComponent) prometheus() *monitoringv1.Prometheus {
 	}
 }
 
+// prometheusHttpApiService sets up a service to open http connection for the prometheus instance
+func (mc *monitorComponent) prometheusHttpApiService() *corev1.Service {
```

Comment:
nit: acronyms should be same-case in golang. So, `prometheusHTTPAPIService` is the right casing here.

Reply:
https://github.com/tigera/operator/pull/1379/commits/4ec097420c2e67725222b5db380aecff96ca1727